src/index.js

```diff
@@ -68,7 +68,8 @@ export default class i18nTransform extends Transform {
       let key = entry.key
       const parts = key.split(this.options.namespaceSeparator)
 
-      if (parts.length > 1) {
+      // make sure we're not pulling a 'namespace' out of a default value
+      if (parts.length > 1 && key !== entry.defaultValue) {
```

Comment:
Don't split if the key is the same as the default value. If the key is `My favorite paragraph: Here I am. I lied. I'm over here!` you'd get a file named, `My favorite paragraph.json` with a key of `{ "Here I am": { "I lied": "I'm over here!" } }` which seems very very wrong.